courses/api_test.py

```diff
@@ -1274,6 +1275,26 @@ def test_generate_program_certificate_success(user, program):
     assert len(ProgramCertificate.objects.all()) == 1
 
 
+def test_force_generate_program_certificate_success(user, program):
+    """
+    Test that generate_program_certificate generate a program certificate
+    """
+    course = CourseFactory.create(program=program)
+    ProgramRequirementFactory.add_root(program)
+    program.add_requirement(course)
+    course_run = CourseRunFactory.create(course=course)
```

Comment:
I've added a fixture but `ProgramRequirementFactory.add_root(program)` is useful in that fixture because I need `courses` to create `course_runs` at multiple places.